Bottom: a1702e8e3d55901898ae87a3c9af39eecf7bf306
Top:    3446ff8fa75c634d15e670c4a2a56cca4d7949a3
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-20 13:45:02 +0800

Refresh of vhost_net-rx-batching

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 1dd31d4..39a2d5a 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -899,7 +899,7 @@ static struct socket *vhost_net_stop_vq(struct vhost_net *n,
 	vhost_net_disable_vq(n, vq);
 	vq->private_data = NULL;
 	while(nvq->rdh != nvq->rdt)
-		kfree_skb(nvq->rxq[rdh++]);
+		kfree_skb(nvq->rxq[nvq->rdh++]);
 	mutex_unlock(&vq->mutex);
 	return sock;
 }
